### DISCUSSION TITLE
Improving the Daily Disk IO Budget remaining graph

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#8195] [Feature Requests]
Hi 👋🏻 

Several members (me included 😅) were confused by the "Daily Disk IO Budget remaining" graphic. We think it is disk usage, and therefore think our instance is 100% used, which is wrong.

I am creating this discussion to see with the community how to improve the graph, or how to avoid this confusion.

**My idea:** change the graph to show what has been used instead of what is remaining.

![image](https://user-images.githubusercontent.com/25107942/183977836-ac06e7bc-63ff-4cb5-9df4-8e6485765e68.png)

**Comment by varna** [comment]
lol, same here, I thought I was capped

**Reply by prilaga** [reply]
The same issue

**Comment by haydn** [comment]
Based on [how it's describe in the docs](https://supabase.com/docs/guides/platform/compute-add-ons), I reckon it'd make more sense for this graph to show disk IO in Mbps and indicate where it's over the threshold for your plan. How much budget you have left isn't as important IMO because it resets every day (and it'd be easy to spot when you hit it because the graph would start to flatline).
